snuba/web/db_query.py

```diff
@@ -466,6 +466,18 @@ def record_cache_hit_type(hit_type: int) -> None:
         "cache_partition_loaded",
         tags={"partition_id": reader.cache_partition_id or "default"},
     )
+
+    cache_wait_timeout: int = int(query_settings.get("max_execution_time", 30))
+    # Increase the cache wait timeout for tiger clusters to debug
+    # ExecutionTimeoutErrors
+    if reader.cache_partition_id and reader.cache_partition_id in {
+        "tiger_errors",
+        "tiger_transactions",
+    }:
+        tiger_wait_timeout_config = state.get_config("tiger-cache-wait-time")
+        if tiger_wait_timeout_config:
+            cache_wait_timeout = tiger_wait_timeout_config
```

Comment:
I'd recommend putting this into a helper function

Reply:
Also something I like to do for code like this is put a note saying "if this code still exists by 2022-05-07, it should be removed, it was a hack to debug something"